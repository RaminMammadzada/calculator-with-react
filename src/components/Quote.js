import React from 'react';

const Quote = () => (
  <>
    <div>
      <h3>Today&apos;s quote: &apos;&apos;Use it, or loose it!&apos;&apos;</h3>
    </div>
  </>
);

export default Quote;
