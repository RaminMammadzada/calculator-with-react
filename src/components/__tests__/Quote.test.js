import ReactTestRenderer from 'react-test-renderer';
import Quote from '../Quote';

it('Creates a snapshot for Quote Page and it should be the same with the previous snapshot', () => {
  const tree = ReactTestRenderer.create(<Quote />).toJSON();

  expect(tree).toMatchSnapshot();
});
