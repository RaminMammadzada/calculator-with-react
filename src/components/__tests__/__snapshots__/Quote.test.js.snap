// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Creates a snapshot for Quote Page and it should be the same with the previous snapshot 1`] = `
<div>
  <h3>
    Today's quote: ''Use it, or loose it!''
  </h3>
</div>
`;
